rules-tests/TypeDeclaration/Rector/Property/TypedPropertyFromStrictConstructorRector/Fixture/skip_not_natively_typed_property_from_data_object.php.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Property\TypedPropertyFromStrictConstructorRector\Fixture;
+
+class NotNativelyTypedDataObject
+{
+    /**
+     * @var string
+     */
+    public $notNativelyTypedProperty;
```

Comment:
could you add fixture for has `string` typed property? This class can be moved to `Source` directory